SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -55,13 +55,14 @@ void ServerWorkerEntityCreator::CreateWorkerEntity()
 
 	AuthorityDelegationMap DelegationMap;
 	DelegationMap.Add(SpatialConstants::GDK_KNOWN_ENTITY_AUTH_COMPONENT_SET_ID, EntityId);
-	Components.Add(AuthorityDelegation(DelegationMap).CreateComponentData());
+	DelegationMap.Add(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID, EntityId);
 
 	if (Settings->CrossServerRPCImplementation == ECrossServerRPCImplementation::RoutingWorker)
 	{
 		Components.Add(ComponentFactory::CreateEmptyComponentData(SpatialConstants::CROSSSERVER_SENDER_ENDPOINT_COMPONENT_ID));
 		Components.Add(ComponentFactory::CreateEmptyComponentData(SpatialConstants::CROSSSERVER_SENDER_ACK_ENDPOINT_COMPONENT_ID));
 	}
+	Components.Add(AuthorityDelegation(DelegationMap).CreateComponentData());
```

Comment:
Any reason for moving this to here?

Reply:
Yes, I will later add a new entry in the if() to delegate authority of some components to the routing worker.